spring-boot/src/main/java/org/springframework/boot/ResourceBanner.java

```diff
@@ -23,10 +23,8 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-
```

Comment:
Apologies. It was my first pull request and didn't realize that my Netbeans IDE performed auto-format while saving the file. Have fixed the above in the fork. I'm not sure what to do next. Do let me know if there's anything else I should be doing.